diff --git a/security/selinux/hooks.c b/security/selinux/hooks.c
index 39410913a694..953317e4c61b 100644
--- a/security/selinux/hooks.c
+++ b/security/selinux/hooks.c
@@ -1319,7 +1319,9 @@ static inline u16 socket_type_to_security_class(int family, int type, int protoc
 			return SECCLASS_SMC_SOCKET;
 		case PF_XDP:
 			return SECCLASS_XDP_SOCKET;
-#if PF_MAX > 45
+		case PF_HSS:
+			return SECCLASS_HSS_SOCKET;
+#if PF_MAX > 46
 #error New address family defined, please update this function.
 #endif
 		}
